jsonschema-core.xml

```diff
@@ -181,17 +181,14 @@
 
             <section title="JSON Document">
                 <t>
-                    A JSON document is an information resource (series of octets) described by the
-                    application/json media type.
-                </t>
-                <t>
-                    In JSON Schema, the terms "JSON document", "JSON text", and "JSON value" are
+                    A JSON document represents the data underlying a given "JSON text" or "JSON value".
+                    Thus in JSON Schema, the terms "JSON document", "JSON text", and "JSON value" are
                     interchangeable because of the data model it defines.
                 </t>
                 <t>
                     JSON Schema is only defined over JSON documents. However, any document or memory
                     structure that can be parsed into or processed according to the JSON Schema data
-                    model can be interpreted against a JSON Schema, including media types like
+                    model can be interpreted against a JSON Schema, including data formats like
```

Comment:
CBOR is a data format. It can have different media types.

Reply:
Good job